frontend/src/metabase/query_builder/components/DataSelector.jsx

```diff
@@ -705,12 +725,19 @@ export class UnconnectedDataSelector extends Component {
   };
 
   render() {
-    const { searchText } = this.state;
+    const { searchText, isSavedQuestionPickerShown } = this.state;
     const { canChangeDatabase, selectedDatabaseId } = this.props;
-    const searchDatabaseId = canChangeDatabase ? null : selectedDatabaseId;
+    const currentDatabaseId = canChangeDatabase ? null : selectedDatabaseId;
 
     const isSearchActive = searchText.trim().length >= MIN_SEARCH_LENGTH;
 
+    const searchPlaceholder = isSavedQuestionPickerShown
+      ? `Search for a question...`
```

Comment:
Just a note: I'm unsure if this adds a new string for the translator (late in the process), but it turns out that we have already a similar string "Search for a question" (without the triple dots). Either we use that (however, it'll be inconsistent with "Searching for a table...") or we use the generic "Search" (this string is already everywhere).
Alternatively, we just keep it as is (with the triple dots) and ask the translators to catch up in 40.1 or something.

Reply:
If this is going in 0.40, we should use the already-translated string, please.

Reply:
I used the existing "Search for a question"